CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 
 ### 1.3.0-alpha.1 (2022-03-25)
 
-- Add filters logical operator prop to source object [#363](https://github.com/CartoDB/carto-react/pull/363)
+- Add filters logical operator prop to source object. This contains a breaking change in the `useSourceFilters` internal function that now returns `{ filters, filtersLogicalOperator }` instead of just `filters` [#363](https://github.com/CartoDB/carto-react/pull/363).
```

Comment:
Thanks Juan. In the changelog we just include the PR name. I'm not sure if this is the best place to add this detail.

Reply:
Yes, I was not very sure where to add this information, but though it needed to be documented. Where shall we place this?

Maybe in the documentation repo ?